complex.c

```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
+
+typedef struct {
+	float real;
+	float imag;
+} cplex;
+
+void display(cplex num)
+{
+	if (num.imag == 0.00)
+		printf("%.2f\n", num.real);
+	else if (num.real == 0.00)
+		printf("%.2fi\n", num.imag);
+}
```

Comment:
I think you're missing conditions to print out the full complex number if neither of its components are 0